python/app.py

```diff
@@ -0,0 +1,26 @@
+import time
+import os
+from urllib.parse import urljoin
+
+from flask import Flask
+import jwt
+
+app = Flask(__name__)
+
+METABASE_SITE_URL = os.environ.get('METABASE_SITE_URL', 'http://localhost:3000')
+METABASE_EMBEDDING_SECRET = os.environ.get('METABASE_EMBEDDING_SECRET')
+METABASE_EMBED_DASHBOARD_ID = int(os.environ.get('METABASE_EMBED_DASHBOARD_ID', '1'))
```

Comment:
Same for here as well. Default of 1 is ok if it should work, otherwise make it required instead like secret.

Reply:
We are now going to be seeding instances with an example dashboard, which I expect to have ID 1, although I don't know if it's 100% guaranteed.